### DISCUSSION TITLE
Is it possible to redirect to a resource from a relation manager

### DISCUSSION THREAD

**Discussion by AnthonySchuijlenburg** [filamentphp/filament#6348] [Help] [answered]
I have two resources which are translatable with the `filament/spatie-laravel-translatable-plugin`. The two relation between the resources is a hasMany relation with a relation manager. In this relation manager I want to link to the edit page of said resource.

The example from the documentation seems to be doing almost what I need: https://filamentphp.com/docs/2.x/tables/actions#single-actions, however, I can not the this to work, since any none of the filament routes are registered anywhere.

Thanks in advance!

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/tricks/relation-manager-edit-record-without-modal

**Reply by AnthonySchuijlenburg** [reply]
That is exactly what I was looking for!
